README.md

```diff
@@ -19,13 +19,40 @@ make build/cli
 A binary will be created in the root directory of the repo, which can be run:
 
 ```
-./cli
+./cluster-service
 ```
 
 
 ### How to use
+```bash
+# set env vars for clusters aws key and secret 
+export AWS_ACCESS_KEY_ID=<key value>
+export AWS_SECRET_ACCESS_KEY=<secret value>
+```
+
+
+```bash
+# run the cleanup command in watch mode to delete persistence resources
+./cluster-service cleanup <cluster_id> --dry-run=false --watch
+# help 
+./cluster-service cleanup --help
+delete aws resources for an rhmi cluster
 
-TODO
+Usage:
+  cluster-service cleanup [cluster id] [flags]
+
+Flags:
+  -d, --dry-run         skip performing actions (default true)
+  -h, --help            help for cleanup
+  -o, --output string   set output format (default "table")
+  -r, --region string   region to delete resources in (default "eu-west-1")
+  -t, --types strings   resource types to cleanup
+  -w, --watch           poll actions being performed indefinitely
+
+Global Flags:
+      --config string   config file (default is $HOME/.cli.yaml)
+      --verbose         enable verbose logging (default is false)
+```
```

Comment:
I feel like this could go out of date pretty fast, can we leave the mention of help but remove the output?

Reply:
:+1: